src/components/PodcastCard/PodcastCard.js

```diff
@@ -1,21 +1,48 @@
 import styled from 'styled-components/macro'
 import PropTypes from 'prop-types'
+import { useState } from 'react'
 
 PodcastCard.propTypes = {
   image: PropTypes.string.isRequired,
   title: PropTypes.string.isRequired,
+  author: PropTypes.string,
+  description: PropTypes.string,
+  categories: PropTypes.object,
 }
 
-export default function PodcastCard({ image, title }) {
+export default function PodcastCard({
+  image,
+  title,
+  author,
+  description,
+  categories,
+}) {
+  const [expanded, setExpanded] = useState(false)
+
   return (
-    <Card>
-      <Image
-        src={image}
-        alt={`Podcast Picture of ${title}`}
-        width="150px"
-        height="150px"
-      />
-      <Title>{title}</Title>
+    <Card onClick={() => setExpanded(!expanded)} clicked={expanded}>
+      <CardMainContainer clicked={expanded}>
+        <Image
+          src={image}
+          alt={`Podcast Picture of `}
+          width="150px"
+          height="150px"
+        />
+        <InnerContainer>
+          <Title>{title}</Title>
+          <Author clicked={expanded}>by {author}</Author>
+        </InnerContainer>
+      </CardMainContainer>
+      {expanded && (
+        <CardExpandContainer>
+          <p>{description}</p>
+          <TagList>
+            {Object.entries(categories)?.map(([key, value]) => (
```

Comment:
Smart 🚀  Nice catch for your possibility of an empty object - ES 2020 rules 🏀

Reply:
Sadly, this doesn't make sense here.
`Object.entries` will always return an array, maybe it's empty, maybe not.
An empty array is truthy, not falsy.

Reply:
I noticed that and actually had to implement the catch for a null object in the PodcastListPage